doc/tutorial.txt

```diff
@@ -75,6 +75,10 @@ The use of a temporary buffer prevents the threads of unexpected wait during
 io access. The "Auto" mode is not implemented yet and fallback to "file" mode
 for the moment.
 
+* logstats: Boolean. Report tasks generated statistics into a logfile. A logfile
+  will be generated for each task, within the logdir, reporting the main metrics
```

Comment:
we already have "logsize": "disable" for that prupose

Reply:
What a horrible naming choice for that parameter! :)

Anyway, it's kind of broken since also with the `disable` option we still generate a logfile, it's just empty :)

Anyway, I guess we still want to go for backward compatibility, thus I will keep this changeset minimal by just fix the empty file generation.

Let's keep instead of the (backward compatible) refactoring of the parameter sematic for the future. ;)